src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const successMessage = `<div class="message">Promise was resolved!</div>`;
+const errorMessage = `<div class="message error-message">Promise was rejected!</div>`;
+
+const successHandler = (message) => {
+  body.insertAdjacentHTML('beforeend', successMessage);
+};
+const errorHandler = (message) => {
+  body.insertAdjacentHTML('beforeend', errorMessage);
+};
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
```

Comment:
You are not resolving the `promise2` promise. You should add a `resolve` function to the `promise2` promise and call it if the `setTimeout` function doesn't call the `reject` function.